build-support-windows/docker.windows.md

```diff
@@ -8,6 +8,7 @@
 - [Dockerfile-consul-dev-windows](#dockerfile-consul-dev-windows)
 - [Dockerfile-fortio-windows](#dockerfile-fortio-windows)
 - [Dockerfile-socat-windows](#dockerfile-socat-windows)
+- [Dockerfile-jaeger-windows](#dockerfile-jaeger-windows)
```

Comment:
let's place whole jaeger documentation, before socat. So it would be sorted by name

Reply:
```suggestion
- [Dockerfile-jaegertracing-windows](#dockerfile-jaegertracing-windows)
```